### ISSUE TITLE
Github integration

### ISSUE THREAD

**Issue by kristiyan-velkov** [mark-nicepants/variables2json-docs#10]
Could you please add github integration so I can be able to commit the json file ?

**Comment by mark-nicepants** [comment]
Yes actively on the roadmap! Don't have a lot of time at the moment to work on this plugin, but hope to resume work on it shortly

**Comment by galgord** [comment]
hi @mark-nicepants i am happy to help do this. I have time and my team would really benefit from this feature.

**Comment by mark-nicepants** [comment]
I'm all up for help, I'll invite you to the repo in a bit. Currently not actively using it myself, so little time at the moment for upgrading it!

**Comment by mark-nicepants** [comment]
Invited you as a collaborator :-)

**Comment by galgord** [comment]
@kristiyan-velkov This was added to the plugin :)
